andi/andi.py

```diff
@@ -7,11 +9,41 @@
 from andi.typeutils import get_union_args, is_union
 
 
+def _get_globalns_as_get_type_hints(func) -> Dict:
+    """ Global namespace resolution extracted from ``get_type_hints`` method.
+     Python 3.7 (https://github.com/python/cpython/blob/3.7/Lib/typing.py#L981-L988) """
```

Comment:
By the way, it is different again in Python 3.8; in this case it mentions  `sys.modules[func.__module__].__dict__`. Would you mind checking how it works in Python 3.8 - are workarounds still needed?

Reply:
Apparently this part of the code is the same in 3.7 and 3.8. I think it is right as it is.